src/prpy/planning/workspace.py

```diff
@@ -62,9 +63,10 @@ def PlanToEndEffectorPose(self, robot, goal_pose, timelimit=5.0,
 
         with robot:
             # Create geodesic trajectory in SE(3)
+            env = robot.GetEnv()
             manip = robot.GetActiveManipulator()
             start_pose = manip.GetEndEffectorTransform()
-            traj = openravepy.RaveCreateTrajectory(self.env, '')
+            traj = openravepy.RaveCreateTrajectory(env, '')
             spec = openravepy.IkParameterization.\
                 GetConfigurationSpecificationFromType(
                     openravepy.IkParameterizationType.Transform6D, 'linear')
```

Comment:
You should wrap `RetimeActiveTrajectory` in a `RobotStateSaver` that preserves the robot's `LinkTransformation`s, since the retimer may modify the robot's active DOF values.